pyproject.toml

```diff
@@ -59,6 +59,7 @@ all = [
     "matplotlib",
 ]
 doc = [
+    "cryptography>=2.0",
```

Comment:
This doesn't look right

Reply:
This is pretty normal. `cryptography` isn't a hard dependency (Scapy has None), so you should be able to load any of the Scapy's module/layers/contrib without having them crashing.

See https://github.com/secdev/scapy/blob/4bc73eda1829627381e8f61cd1ab4e2d7bebf524/scapy/config.py#L1164 and everywhere it's used. Usually the import is inside a try:catch, then we have that decorator for each function it's used in.